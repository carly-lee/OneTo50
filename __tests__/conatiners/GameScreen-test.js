import 'react-native';
import React from 'react';
import GameScreen from 'containers/gamescreen';
import renderer from 'react-test-renderer';

describe( 'GameScreen', ()=>{
  it( 'renders correctly',()=>{
    const tree = renderer.create( <GameScreen /> );
  });
});