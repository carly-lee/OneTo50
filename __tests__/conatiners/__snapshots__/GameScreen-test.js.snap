exports[`GameScreen renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 15,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "skyblue",
        "flex": 0.2,
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "pink",
          "flex": 0.6,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        0
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 0.4,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        1
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "grey",
        "flex": 0,
        "flexDirection": "row",
        "flexWrap": "wrap",
        "height": 350,
        "width": 350,
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "opacity": 1,
          "width": 70,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        1
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        2
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        3
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        4
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        5
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        6
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        7
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        8
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        9
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        10
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        11
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        12
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        13
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        14
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        15
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        16
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        17
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        18
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        19
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        20
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        21
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        22
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        23
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        24
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "white",
          "borderWidth": 1,
          "flex": 0,
          "flexDirection": "column",
          "height": 70,
          "justifyContent": "center",
          "width": 70,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
          }
        }>
        25
      </Text>
    </View>
  </View>
</View>
`;
